lib/features/contextPad/ImprovedContextPadProvider.js

```diff
@@ -176,7 +176,7 @@ ImprovedContextPadProvider.$inject = [ 'contextPad', 'eventBus', 'popupMenu' ];
 
 // helpers //////////
 function getIcon(icon) {
-  return `<div class="entry">${icons[icon]}</div>`;
+  return `<div class="entry" draggable="true">${icons[icon]}</div>`;
```

Comment:
Shouldn't we add `draggable="true"` only to the icons that support drag to be semantically correct?

Reply:
That makes total sense. Updated